### ISSUE TITLE
src/locales/*/address/street.ts is implausible

### ISSUE THREAD

**Issue by ST-DDT** [faker-js/faker#1472]
https://github.com/faker-js/faker/blob/79858fea203bce7ada9e9bcc7751f6ab25123977/src/locales/fr/address/street.ts#L1

The file should either be deleted or replaced with the English one.

**Note:** After #1469 is merged these files will then be located under `src/locales/*/location/city.ts`.

**Comment by ST-DDT** [comment]
On hold till #1469 is fixed.

**Comment by ST-DDT** [comment]
https://github.com/faker-js/faker/blob/79858fea203bce7ada9e9bcc7751f6ab25123977/src/locales/fr_BE/address/street.ts#L4

**Comment by ST-DDT** [comment]
https://github.com/faker-js/faker/blob/79858fea203bce7ada9e9bcc7751f6ab25123977/src/locales/fr_CH/address/street.ts#L1

**Comment by ST-DDT** [comment]
https://github.com/faker-js/faker/blob/79858fea203bce7ada9e9bcc7751f6ab25123977/src/locales/pt_PT/address/street.ts#L1-L7

**Comment by ST-DDT** [comment]
I might have missed some files, so the other `street.ts` files should be checked again.

**Comment by xDivisionByZerox** [comment]
Resolved by #2051.
